billit/client.py

```diff
@@ -35,9 +36,13 @@ def __init__(self, api_key, environment=PRODUCTION_ENVIRONMENT):
             self.base_url = SANDBOX_BASE_URL
 
     def _handle_response(self, response):
+
         if response.status_code == 401:
             raise AuthenticationError(response.json()["message"], response.status_code)
 
+        if response.status_code == 204:
+            return response.text
```

Comment:
Let's prefer an empty dictionary or None here, otherwise the return type is different in this case (string).

```suggestion
            return None
```